README.md

```diff
@@ -1,28 +1,121 @@
-# SubQuery - Dictionary 
+# 🤔 Introduction
 
-This special SubQuery Project provides a dictionary of data that pre-indexes events on chain to dramatically improve indexing the performance of your own SubQuery Project, sometimes up to 10x faster.
+The Dictionary provides a glossary of data that pre indexes On-Chain events, drastically improving the overall Indexing performance. Sometimes by a factor of 10, which is quite significant.
```

Comment:
We can say:

Ternoa Dictionary records all the native subtrate on-chain data of the Ternoa blockchain: blocks, extrinsics and events. It is a glossary of data that pre indexes chain events, drastically improving the overall indexing performance.

Reply:
Done